index.html

```diff
@@ -8,6 +8,57 @@
     <link rel="stylesheet" href="css/styles.css" />
   </head>
   <body>
+    <div class="wrapper">
+      <header>
+        <nav>
+          <ul>
+            <li><i class="fab fa-apple"></i></li>
+            <li>Store</li>
+            <li>Mac</li>
+            <li>iPod</li>
+            <li>iPhone</li>
+            <li>iPad</li>
+            <li>iTunes</li>
+            <li>Support</li>
+            <li>
+              <label>
+                <i class="fab fa-search"></i>
+                <input type="text" />
+              </label>
+            </li>
+          </ul>
+        </nav>
+        <div>
+          <img id="verse" src="assets/verse.png" alt="verse" />
+          <img id="share" src="assets/share.png" alt="share" />
+          <img id="watch" src="assets/watch.png" alt="watch" />
+        </div>
+      </header>
+      <main>
+        <img class="products" src="assets/iphone5s.jpg" alt="iphone5s" />
```

Comment:
These images should be wrapped with an ```<a> tag``` The images are links (to another articles or #)